src/main/java/com/bergerkiller/bukkit/tc/TrainCarts.java

```diff
@@ -129,6 +132,15 @@ public PathProvider getPathProvider() {
         return this.pathProvider;
     }
 
+    /**
+     * Gets the Economy manager
+     *
+     * @return
+     */
+    public static Economy getEconomy() {
```

Comment:
I don't know about this being static, I feel like it shouldn't since plugin itself is already static.

Reply:
Agreed, generally I leave such things local and access them using Traincarts.plugin.getEconomy(). Ideally I would send an instance of Traincarts along

Reply:
I changed this in commit 75ea6594